polkadot/runtime/rococo/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,544 @@
+		get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+		get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+		get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+	])
+}
+
+fn rococo_session_keys(
+	babe: BabeId,
+	grandpa: GrandpaId,
+	para_validator: ValidatorId,
+	para_assignment: AssignmentId,
+	authority_discovery: AuthorityDiscoveryId,
+	beefy: BeefyId,
+) -> SessionKeys {
+	SessionKeys { babe, grandpa, para_validator, para_assignment, authority_discovery, beefy }
+}
+
+fn default_parachains_host_configuration(
+) -> runtime_parachains::configuration::HostConfiguration<primitives::BlockNumber> {
+	use primitives::{
+		vstaging::node_features::FeatureIndex, AsyncBackingParams, MAX_CODE_SIZE, MAX_POV_SIZE,
+	};
+
+	runtime_parachains::configuration::HostConfiguration {
+		validation_upgrade_cooldown: 2u32,
+		validation_upgrade_delay: 2,
+		code_retention_period: 1200,
+		max_code_size: MAX_CODE_SIZE,
+		max_pov_size: MAX_POV_SIZE,
+		max_head_data_size: 32 * 1024,
+		max_upward_queue_count: 8,
+		max_upward_queue_size: 1024 * 1024,
+		max_downward_message_size: 1024 * 1024,
+		max_upward_message_size: 50 * 1024,
+		max_upward_message_num_per_candidate: 5,
+		hrmp_sender_deposit: 0,
+		hrmp_recipient_deposit: 0,
+		hrmp_channel_max_capacity: 8,
+		hrmp_channel_max_total_size: 8 * 1024,
+		hrmp_max_parachain_inbound_channels: 4,
+		hrmp_channel_max_message_size: 1024 * 1024,
+		hrmp_max_parachain_outbound_channels: 4,
+		hrmp_max_message_num_per_candidate: 5,
+		dispute_period: 6,
+		no_show_slots: 2,
+		n_delay_tranches: 25,
+		needed_approvals: 2,
+		relay_vrf_modulo_samples: 2,
+		zeroth_delay_tranche_width: 0,
+		minimum_validation_upgrade_delay: 5,
+		async_backing_params: AsyncBackingParams {
+			max_candidate_depth: 3,
+			allowed_ancestry_len: 2,
+		},
+		node_features: bitvec::vec::BitVec::from_element(
+			1u8 << (FeatureIndex::ElasticScalingMVP as usize),
+		),
+		scheduler_params: SchedulerParams {
+			lookahead: 2,
+			group_rotation_frequency: 20,
+			paras_availability_period: 4,
+			..Default::default()
+		},
+		..Default::default()
+	}
+}
+
+#[test]
+fn default_parachains_host_configuration_is_consistent() {
+	default_parachains_host_configuration().panic_if_not_consistent();
+}
+
+fn rococo_testnet_genesis(
+	initial_authorities: Vec<(
+		AccountId,
+		AccountId,
+		BabeId,
+		GrandpaId,
+		ValidatorId,
+		AssignmentId,
+		AuthorityDiscoveryId,
+		BeefyId,
+	)>,
+	root_key: AccountId,
+	endowed_accounts: Option<Vec<AccountId>>,
+) -> serde_json::Value {
+	let endowed_accounts: Vec<AccountId> = endowed_accounts.unwrap_or_else(testnet_accounts);
+
+	const ENDOWMENT: u128 = 1_000_000 * ROC;
+
+	serde_json::json!({
```

Comment:
If you want to be really pedantic about these refactors, you can check the output of this against a previous output that is generated from the node side and assert that while migrating to using runtime presets, everything remains backwards compatible.

Reply:
This was already done when migrating to `json!` format. Test were added in #1256 and remove in this commit: https://github.com/paritytech/polkadot-sdk/pull/1256/commits/f4d3f32cb8c974a3b6b06ca8248daa99f3d0266e